src/proj.h

```diff
@@ -0,0 +1,205 @@
+/******************************************************************************
```

Comment:
I'm not sure how much we care about that but it would seem that /usr/include/proj.h is(was?) a SGI include file... https://www.google.fr/?gws_rd=ssl#q=%22%2Fusr%2Finclude%2Fproj.h%22